examples/nft-nd-nns/nns.go

```diff
@@ -353,6 +360,15 @@ func Resolve(name string, typ RecordType) string {
 	return resolve(ctx, name, typ, 2)
 }
 
+// GetAllRecords returns an Iterator with RecordState items for given name.
+func GetAllRecords(name string) iterator.Iterator {
```

Comment:
Don't we want to add the corresponding API to RPC client?

Reply:
Fixed.